### DISCUSSION TITLE
Tagging a user on Bitbucket

### DISCUSSION THREAD

**Discussion by binary64** [danger/danger-js#1151] [Q&A]
Hi, I wish to tag a user "@Username" on danger's Bitbucket comments.

I've tried these but no luck

```
@{username}
@{guid}
```

But it comes out malformed:

```
@
```
![image](https://user-images.githubusercontent.com/1680627/131764594-4adf959a-c073-4ebc-be70-b3dfeb825783.png)

**Comment by orta** [comment]
I'm unsure, I'd recommend making a test and seeing where it fails 👍🏻
